util/site/blocks.json

```diff
@@ -227,8 +227,8 @@
         "href": "/hw/top_earlgrey/ip/ast#top",
         "report": null
     },
-    "padding": {
-        "name": "Padding",
+    "padring": {
+        "name": "Padring",
```

Comment:
Can this commit be squashed so that this "padring" is not introduced in the commit history?